extensions/sql-bindings/src/sql-bindings.d.ts

```diff
@@ -34,19 +38,22 @@ declare module 'sql-bindings' {
 		createAzureFunction(): Promise<void>;
 
 		/**
-		 * Prompts the user to select type of binding and returns result or undefined if the user cancelled out of the prompt
-		 * @param funcName (Optional) Name of the function we are adding the SQL Binding to
+		 * Prompts the user to select type of binding and returns result
+		 * @param objectType (Optional) The type of object user choose to insert/upsert into
```

Comment:
It's preferable to indicate what the default value is (or what happens) when optional values are left out. Otherwise that's basically undefined behavior which makes it more difficult for callers to know what to expect.